lib/cubit/task_state.dart

```diff
@@ -30,14 +30,14 @@ class TaskState extends Equatable {
 
 //make into one cubit ^
```

Comment:
Please change this to:

```suggestion
// TODO - make into one cubit ^
```

VS Code will track any comments with the text "TODO" in them

Reply:
I think this one was missed. This is a great TODO comment. It just needs the "TODO"

Reply:
Oh gotcha, yeah I missed this one